doc/en/components/dock-manager-electron.md

```diff
@@ -0,0 +1,136 @@
+```
+
+In the `paneHeaderDragEnd` function we detect if the pane was dropped outside the browser/application window and we call the `droppedOutOfWindow` function.
+
+```ts
+const paneHeaderDragEnd = async (event: DragEvent) => {
+    event.preventDefault();
+    // ...
+
+    // dropped outside of the browser
+    if (event.dataTransfer.dropEffect === 'none') {
+        await droppedOutOfWindow(event);
+    }
+    // ...
+}
+```
+
+When the pane header is dropped inside a document we call the [`dropPane`]({environment:infragisticsBaseUrl}/products/ignite-ui/dock-manager/docs/typescript/latest/interfaces/igcdockmanagercomponent.html#droppane) method which notifies the Dock Manager that the dragged pane was dropped. If the pane was dropped on a docking indicator the method returns `true`. If the pane was dropped in the same window it was dragged from, the pane will be docked to its new position automatically. However, if it was dropped in another window we call the `droppedInAnotherWindow` function.
+
+```ts
+const handleDocumentDrop = async (event: DragEvent) => {
+    const contentId = (dockManager.draggedPane as IgcContentPane).contentId;
+
+    const docked = await dockManager.dropPane();
+
+    if (docked) {
+        const contentElement = dockManager.querySelector('[slot=' + contentId + ']');
+        
+        // if the content element is missing from the current dock manager it means it comes from another window
+        if (!contentElement) {
+            await droppedInAnotherWindow();
+        }
+    }
+}
+```
+
+When a pane is dropped out of window or in another window we remove the `draggedPane` from its current Dock Manager component.
+
+```ts
+const draggedPane = dockManager.draggedPane as IgcContentPane;
+await dockManager.removePane(draggedPane);
+dockManager.layout = { ...dockManager.layout };
+```
+
+This updates only the `layout` of the Dock Manager. We should also move the pane content element to its new window. For this purpose, we use the [`document.adoptNode()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/adoptNode) method.
+
+```ts
+const contentElement = dockManager.querySelector('[slot=' + draggedPane.contentId + ']');
+const newDocument = childWindow.document;
+const newDockManager = newDocument.getElementById('dockManager') as IgcDockManagerComponent;
+const adoptedNode = newDocument.adoptNode(contentElement);
+newDockManager.appendChild(adoptedNode);
+```
+
+We are using the native [`window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) method to open a new window in the Renderer process. We set the `nativeWindowOpen` option to `true` when creating the `BrowserWindow` in the `index.ts`. This gives us a direct access to the child `Window` object and its `document`. You could read more about opening windows from the Renderer process in this Electron [topic](https://www.electronjs.org/docs/api/window-open). Please note that the `nativeWindowOpen` option is still experimental.
```

Comment:
This paragraph is a little disconnected from the previous one but I don't have a better suggestion.